SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+## Supported Versions
+
+We support any version published via our official distribution channels.
```

Comment:
I suggest that we only ensure that the Chrome and Firefox versions can be fixed as soon as possible, because Edge and Safari have relatively large API changes and may have technical limitations.

Reply:
Before the big update a few years ago Edge was very different, but now it is almost the same as Chrome. I might just be forgetting, but what are the big differences in Edge? Given that we have 2M+ active users there, we should support it. For Safari I agree with you.

Reply:
If we move forward with not supporting safari, we should indicate it is a "community version" or similar in the README.